parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -374,6 +378,14 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+match_types! {
+	pub type RelayOrSystemParachains: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 0, interior: X1(Parachain(kusama_runtime_constants::system_parachain::STATEMINE_ID |
```

Comment:
So this means we don't collect any fees if the outgoing XCM message is destined for itself? Not sure if this is a good idea, because people can send a lot of messages to loop back to Statemine and block the queue.